multiqc/modules/ngsderive/ngsderive.py

```diff
@@ -455,3 +447,80 @@ def add_junctions_data(self):
             [the documentation](https://stjudecloud.github.io/ngsderive/subcommands/junction_annotation/).""",
             plot=bargraph.plot([bardata, bardata], cats, pconfig),
         )
+
+    def add_endedness_data(self):
+        # Write data to file
+        self.write_data_file(self.endedness, "ngsderive_endedness")
+
+        general_data = {}
+        rpt_present = False
+        for sample, endedness_data in self.endedness.items():
+            general_data[sample] = {
+                "endedness": endedness_data.get("Endedness"),
+            }
+            if "Reads per template" in endedness_data:
+                rpt_present = True
+
+        general_headers = OrderedDict()
+        general_headers["endedness"] = {
+            "title": "Predicted Endedness",
+            "description": "Predicted library endedness from ngsderive",
+        }
+        self.general_stats_addcols(general_data, general_headers)
+
+        headers = OrderedDict()
+        headers["FyLn"] = {
+            "title": "f+l- count",
+            "description": "Number of reads with the 'first in template' FLAG set and the 'last in template' FLAG unset",
+            "format": "{:,d}",
```

Comment:
Should use `"shared_key": "read_count"` and the appropriate number base modifiers. See [docs](https://multiqc.info/docs/reports/customisation/#number-base-multiplier).

Hmm, docs are a bit thin on this. Example here:

https://github.com/ewels/MultiQC/blob/57f3588c5a524f855b0706022e6db44f423588d1/multiqc/modules/ivar/ivar.py#L110-L116

Reply:
Would this automatically scale the values? As I shared in another comment, these data are being checked for exact equality. Any scaling would mask problems users should know about. The value of the "one's" place is important, as off-by-one errors are the most common class of error.

Reply:
I think that for readability, decimal separators do the job, so there is no need to use read count multipliers that hide the precision that's important here. Unless we add another helpful metric like the difference between f+l- and f-l+. Though I'm not sure if MultiQC should be creative in this case.

Reply:
ok 👍🏻 

Worth considering a custom shared key anyway, to preserve bar scaling across columns? I _think_ that `read_count` might do some magic with modifiers (?) but a custom string shouldn't.